docs/source/howto/workflows.rst

```diff
@@ -18,7 +18,157 @@ Writing workflows
 Running a predefined workflow
```

Comment:
Sorry to come back on this now, but I was just wondering whether we should renaming this to "Launching a predefined workflow". I have been trying to strictly use "launch" when talking in the generic sense, since "run" and "submit" will quickly get specific meaning when talking in the context of AiiDA, given that the two launchers  `run` and `submit` fundamtenally change the behavior.

I think "Launching" is a common enough term for novice users to understand straight away what it means (instead of running) and it will already keep consistency with the language that will follow

Reply:
Good point! Everyone should understand what "launching" means, and this will make the terminology in the docs more consistent. Do you agree, @csadorf?

Reply:
I think that's a good idea! 👍